### ISSUE TITLE
Proposal: Change to Database folder structure and naming

### ISSUE THREAD

**Issue by rhosking** [The-Academic-Observatory/observatory-platform#166]
Currently the database folder is split into analysis and platform, like:

database
----> analysis
--------> bigquery
------------> model
------------> schema
----> platform
--------> model
--------> schema

To better align with the naming, and to remove one level I propose a change to the analysis database structure (removing the bigquery folder), and the addition to two other categories of data (telescope and service). The analysis folder will contain the schemas for the dervied and aggregated tables, while the telescopes folder will match with the telescopes/dags. The addition of the 'services' folder provides a location for future work.

database
----> telescopes
--------> model
--------> schema
----> analysis
--------> model
--------> schema
----> platform
--------> model
--------> schema
----> services
--------> model
--------> schema

In summary:
- [ ] changes to folder structure
- [ ] move all the current telescope related schemas currently in database->analysis->bigQuery>schema to the telescopes->schema folder
- [ ] removing the '_schema' from the existing schema files
- [ ] standardise the lowercase + underscore syntax
- [ ] crossref_schema.json changed to crossref_metadata.json
- [ ] grid_old_schema.json changed to grid_pre_2015.json
